Source/WebCore/crypto/gcrypt/CryptoAlgorithmEd25519GCrypt.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2022 Apple Inc. All rights reserved.
+ * Copyright (C) 2022 Metrological Group B.V.
+ * Copyright (C) 2022 Igalia S.L.
```

Comment:
Is it normal?

Reply:
What do you mean?

Reply:
It is a new file so only Apple Copyright should be there.